platforms/org.eclipse.mita.platform.xdk110/1.0.0/xdk110.platform

```diff
@@ -980,6 +980,27 @@ bus named-singleton GPIO {
 	signal digitalOut(pin : GPIOPin, initialValue : bool = false) : bool
 }
 
+export exception EndOfFileException
+
+io named-singleton SDCard {
+	generator "org.eclipse.mita.platform.xdk110.io.SDCardGenerator"
+	sizeInferrer "org.eclipse.mita.platform.xdk110.io.SDCardSizeInferrer"
+
+	signal resumingFile_Text_Read(filePath: string, blockSize: uint32) : string
+	signal appendingFile_Text_Write(filePath: string) : string
+
+	signal rewindingFile_Text_Read(filePath: string, fileSize: uint32) : string
+	signal rewindingFile_Text_Write(filePath: string) : string
+
+	signal resumingFile_Binary_Read(filePath: string, blockSize: uint32) : array<uint8>
+	signal appendingFile_Binary_Write(filePath: string) : array<uint8>
+
+	signal rewindingFile_Binary_Read(filePath: string, fileSize: uint32) : array<uint8>
+	signal rewindingFile_Binary_Write(filePath: string) : array<uint8>
+
+	modality SDCardAvailable
```

Comment:
Modalities should start with a lowercase character: `sdCardAvailable`

Reply:
actually that modality isn't even implemented so I'll remove it. Even if it were implemented currently `SDCard_Enable` fails if no SD card is inserted so the implementation would be `*_result = true; return RETCODE_OK`, which doesn't help at all.